### PR TITLE
Verb errors documented; object errors handling implemented

### PR DESCRIPTION
**Pull request by Holado**
Basis for pull request for branch objcheck in GreynirCorrect

* Verb error categories have been documented in Verbs.conf
* Object errors are now read correctly into data structures (and handled correctly in GreynirCorrect)
* Groundwork for other error types laid

### CODE REVIEW

**Comment by Holado** [comment]
Pytest works for me, I don't see the problem with the checks...

File: src/reynir/verbframe.py [hunk 1]
```diff
@@ -322,17 +328,19 @@ def create_from_config(cls, s: str) -> None:
...
-                cls.CASE_FRAMES[case_key].append(vf)
-            # Add to the dictionary of all verb frames
-            cls.ALL_FRAMES[vf.key].append(vf)
-            # Add to the set of known verb lemmas
-            cls.VERBS.add(verb)
+        # Create a VerbFrame instance
+        # Note: In order to parse verbs with wrong arguments,
+        # the frame needs to be present as a regular VerbFrame instance
+        # that is then marked as an error in GreynirCorrect
+        vf = cls(verb, args, prepositions, particle, score)
```
Comment (vthorsteinsson) [inline src/reynir/verbframe.py:335]: We need to discuss this - GreynirPackage should not allow a parse with a wrong argument, but GreynirCorrect should. This is normally effected by having the latter override standard behavior in the former, e.g. via a custom parser class (like GreynirCorrect implements).

Comment (Holado) [reply src/reynir/verbframe.py:335]: We have ruminated on this before but hadn't reached a conclusion. Allowing known, wrong structures (with a severe score reduction) would make GreynirPackage more robust, and could help unparsable sentences. 
As we have another solution up our sleeve, I'll look into how the error grammar rules are excluded except in GreynirCorrect and try to imitate that.

**Comment by Holado** [comment]
This should be ready for merging at the same time as objcheck in GreynirCorrect

File: src/reynir/verbframe.py [hunk 2]
```diff
@@ -325,22 +326,27 @@ def create_from_config(cls, s: str) -> None:
...
         # Add to verb database
+        vf = cls(verb, args, prepositions, particle, score)
+        case_key = vf.case_key
         if error:
             # Add this to the error database
             VerbErrors.add_error(verb, args, prepositions, particle, error)
-        # Create a VerbFrame instance
         # Note: In order to parse verbs with wrong arguments,
-        # the frame needs to be present as a regular VerbFrame instance
+        # the frame needs to be present as an extra VerbFrame instance
```
Comment (vthorsteinsson) [inline src/reynir/verbframe.py:335]: Fix indent of this comment (should be indented 1 additional level)

File: src/reynir/verbframe.py [hunk 3]
```diff
@@ -353,6 +359,11 @@ def matches_arguments(cls, verb_with_cases: str) -> bool:
             with its configured arguments? """
         return verb_with_cases in cls.CASE_FRAMES
 
+    @classmethod
+    def matches_error_arguments(cls, verb_with_cases: str) -> bool:
+        """ Does the given key, match any erroneous verb frames? """
```
Comment (vthorsteinsson) [inline src/reynir/verbframe.py:364]: Delete comma in comment

File: src/reynir/verbframe.py [hunk 4]
```diff
@@ -353,6 +359,11 @@ def matches_arguments(cls, verb_with_cases: str) -> bool:
...
+        return verb_with_cases in cls.WRONG_CASE_FRAMES
```
Comment (vthorsteinsson) [inline src/reynir/verbframe.py:365]: This looks suspicious - cls.WRONG_CASE_FRAMES is not a list of strings, it's a list of VerbFrame instances.

File: src/reynir/verbframe.py (hunk 2 above)
Comment (Holado) [reply src/reynir/verbframe.py:335]: Done

File: src/reynir/verbframe.py (hunk 3 above)
Comment (Holado) [reply src/reynir/verbframe.py:364]: Done

File: src/reynir/verbframe.py (hunk 4 above)
Comment (Holado) [reply src/reynir/verbframe.py:365]: The key in cls.WRONG_CASE_FRAMES is a string, this checks if verb_with_cases is a key. Changed to cls.WRONG_CASE_FRAMES.keys() to make this clearer

**Pull request merged**